keras_cv/layers/preprocessing/random_crop.py

```diff
@@ -112,10 +95,6 @@ def _resize(self, image):
     def augment_label(self, label, transformation, **kwargs):
         return label
 
-    def augment_bounding_boxes(self, bounding_boxes, transformation=None, **kwargs):
-        # Divya to-do add support in follow up PR
-        return bounding_boxes
-
     def compute_output_shape(self, input_shape):
```

Comment:
One other bit -- I think we should override `compute_image_signature` instead -- I don't think this method does anything in our Base layer